commitment/commitment_test.go

```diff
@@ -342,28 +332,41 @@ func TestMintTapCommitment(t *testing.T) {
 	for _, testCase := range testCases {
 		success := t.Run(testCase.name, func(t *testing.T) {
 			details := testCase.f()
-			// Test cases with a nil amount should fail in the call
-			// to Mint() below, not on asset creation. Use the safe
-			// amount of 1 here, and pass the original amount to
-			// Min() below.
-			var amt uint64
-			if details.Amount == nil {
-				amt = 1
-			} else {
-				amt = *details.Amount
-			}
 
-			// TOO(jhb): fix nil amount issues, use this err
-			protoAsset, err := asset.New(
-				testCase.g, amt, details.LockTime,
-				details.RelativeLockTime,
-				asset.NewScriptKeyBip86(details.ScriptKey), nil,
-			)
+			// Test case concerns amount sanity checking, skip
+			// group key creation.
+			var err error
+			amt := details.Amount
+			invalidNormalAmt := details.Type != asset.Collectible &&
```

Comment:
We could create two package-level variables `zero` and `one` to make it easier to use with the collectible amounts.